_episodes/07-integration.md

```diff
@@ -17,7 +19,7 @@ the gears and cogs that make up the system. On their own, they can be of the hig
 quality. Unit tests verify that each gear is well made. However, the clock still
 needs to be put together. The gears need to fit with one another.
 
-> ## Telling The Time {.callout}
+> ## Further explanation
```

Comment:
Why all of these changes of the titles? I prefer Telling the Time. It refers to the integration test as a clock made up of units. Why do you feel "Further Explanation" is better?